onebusaway-api-webapp/src/main/java/org/onebusaway/api/actions/api/gtfs_realtime/TripUpdatesForAgencyAction.java

```diff
@@ -75,6 +82,18 @@ protected void fillFeedMessage(FeedMessage.Builder feed, String agencyId,
         vehicleDesc.setId(normalizeId(vehicle.getVehicleId()));
       }
     }
+    if(feedTimestamp != 0){
+      long lastModifiedMills = feedTimestamp * 1000L;
+      String lastModifiedHeader = _sdf.format(new Date(lastModifiedMills));
+      HttpServletResponse response = ServletActionContext.getResponse();
+      response.setHeader("Last-Modified", lastModifiedHeader);
+    } else {
+      long lastModifiedMills = timestamp * 1000L;
+      String lastModifiedHeader = _sdf.format(new Date(lastModifiedMills));
+
+      HttpServletResponse response = ServletActionContext.getResponse();
+      response.setHeader("Last-Modified", lastModifiedHeader);
+    }
```

Comment:
If this is the same code as above, can it be a method that's inherited from GtfsRealtimeActionSupport?

Reply:
Yes, It can be and it is also the best practice.modified accordingly.